spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractMessageChannelBinder.java

```diff
@@ -110,6 +115,9 @@ public AbstractMessageChannelBinder(boolean supportsHeadersNatively, String[] he
 			if (e instanceof BinderException) {
 				throw (BinderException) e;
 			}
+			else if (e instanceof ProvisioningException) {
+				throw (ProvisioningException) e;
```

Comment:
M-m-m. How this exception can be possible here if we have done provisioning already a bit before via ` this.provisioningProvider.provisionProducerDestination()`?
Maybe the idea was to include that call to the `try...catch` as well?
Thanks